docs/pages/preview/custom-build-config-schema.mdx

```diff
@@ -0,0 +1,280 @@
+
+```yaml
+build:
+  name: Demo
+  steps:
+    - run:
+        name: List assets
+        # @info #
+        working_directory: assets
+        # @end #
+        command: ls -la
+```
+
+#### Use a single step
+
+For example, a workflow with the following `steps` will run a single command to install npm dependencies:
+
+```yaml
+build:
+  name: Install npm dependencies
+  steps:
+    # @info #
+    - run:
+        name: Install dependencies
+        command: npm install
+    # @end #
+```
+
+> **info** `-` before `run` counts as indentation.
+
+#### Use multiple steps
+
+When multiple `steps` are defined, they are executed sequentially. For example, a workflow with the following `steps` will first run a single command to install npm dependencies and then run a command to run tests:
+
+```yaml
+build:
+  name: Run tests
+  steps:
+    # @info #
+    - run:
+        name: Install dependencies
+        command: npm install
+    - run:
+        name: Run tests
+        command: |
+          echo "Running tests..."
+          npm test
+    # @end #
+```
+
+## `functions`
+
+Defined to describe a reusable function or [built-in EAS functions](#built-in-eas-functions) that can be used in a workflow. All config options to create a function are specified with the following properties:
+
+### `name`
+
+The name of a function that is used in a `build`. It is also used to identify the function triggered in the build logs. For example, a function named `greetings` is created:
+
+```yaml
+functions:
+  greetings:
+    # @info #
+    name: Say Hi!
+    # @end #
+```
+
+### `platforms`
+
+Name of the platform that a specific function should get executed. `linux` is used for Android and `darwin` for iOS. If not specified, the function is executed for both platforms when the appropriate build runs for each platform.
+
+### `inputs`
+
+An input value is provided to a function.
+
+#### `inputs.default_value`
+
+You can use `default_value` to provide one default input. For example, a function has a default value of `Hello world`:
+
+```yaml
+functions:
+  greetings:
+    name: Say Hi!
+    inputs:
+      - name: name
+        # @info #
+        default_value: Hello world
+        # @end #
+```
+
+#### `inputs.allowed_values`
+
+You can use `allowed_values` to provide multiple values in an array. For example, a function has multiple allowed values:
+
+```yaml
+functions:
+  greetings:
+    name: Say Hi!
+    inputs:
+      - name: name
+        default_value: Hello world
+        # @info #
+        allowed_values: [Hi, Hello, Hey]
+        # @end #
+```
+
+### `outputs`
+
+An output value is expected from a function. For example, a function has an output value of `Hello world`:
+
+```yaml
+functions:
+  greetings:
+    name: Say Hi!
+    # @info #
+    outputs: [value]
+    # @end #
+    command: set-output value Hello
```

Comment:
If I understand the line above (https://github.com/expo/expo/pull/22165/files#diff-7e91ba88739076f4e473a23a6f5413419ac30c5694ba4fe3d0fc9e84c52c496fR184) correctly, this has to be:
```suggestion
    command: set-output value "Hello world"
```